wp-content/themes/haruoshi_origin/src/scss/header.scss

```
@@ -6,7 +6,7 @@
 	position: fixed;
 	inset: 0;
 	margin: 0 auto;
-	max-width: 840px;
+	max-width: $contents-max-width;
```

ここの指定あんまよくない気がしてきた
マークアップの構造を
```
header.site-header
  div.site-header__contents
    h1.site-header__logo
    nav
      ul.global-nav
```
にして
```suggestion
	width: 100%;
```
からの
```diff
.site-header__contents {
  max-width: $contents-max-width;
}
```
がいいかな

---

これにしたら壊れちゃったけど...

---

あ、嘘かも

---

これheightはどうしたらいいの？
親のままだと中身潰れちゃって、contentsにうつすとheaderばかでかくなる

---

height指定は.site-headerに書いたままで
```
.site-header {
+  align-items: center;
}
```
とかでなおらない？

---

done https://github.com/oooshima/origin_wordpress/pull/25/commits/661a6b878b24dcab5acaf77d83b0267d2f71db73